client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Why disabling this rule. Just yarn under the `client` folder to resolve this error.